cfme/tests/services/test_provision_stack.py

```diff
@@ -127,7 +131,16 @@ def catalog():
     catalog = Catalog(name=cat_name, description="my catalog")
     catalog.create()
     yield catalog
-    catalog.delete()
+
+
+@pytest.fixture(scope="function")
+def set_grid_view():
```

Comment:
I think there is already a function for setting the gridview as the default, please take a look in the configure module